src/platform/arguments.c

```diff
@@ -0,0 +1,109 @@
+#include "arguments.h"
+
+#include "SDL.h"
+
+#include <stdio.h>
+
+#define CURSOR_SCALE_ERROR_MESSAGE "Option --cursor-scale must be followed by a scale value of 1, 1.5 or 2\n\n"
+#define DISPLAY_SCALE_ERROR_MESSAGE "Option --display-scale must be followed by a scale value between 0.5 and 5\n\n"
+
```

Comment:
add:
#define UNKNOWN_OPTION_ERROR_MESSAGE "The option %s is not understood\n\n"